docs/source/user_guide.rst

```diff
@@ -0,0 +1,14 @@
+==========
+User Guide
+==========
+
+In the guides below, a variety of topics are covered so you can learn how to use Compose as productive as possible.
```

Comment:
I think this sentence could be more to the point

Reply:
Hmm, what about this?
> In the guides below, you will learn how to use label transforms and more to help you generate training examples more effectively.